app-modules/assist-data-model/src/Policies/StudentPolicy.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Assist\AssistDataModel\Policies;
+
+use App\Models\User;
+use Illuminate\Auth\Access\Response;
+use Assist\AssistDataModel\Models\Student;
+
+class StudentPolicy
```

Comment:
Because these are never things a user would modify, create, or delete themselves, let's only generate the view and view any permissions.

See Role.